Classes/Service/MauticAuthorizeService.php

```diff
@@ -265,4 +266,79 @@ protected function translate(string $key): string
         }
         return $this->languageService->sL('LLL:EXT:mautic/Resources/Private/Language/locallang_mod.xlf:' . $key);
     }
+
+    public function validateAccessToken(): bool
+    {
+        if ($this->extensionConfiguration['authorizeMode'] === YamlConfiguration::OAUTH1_AUTHORIZATION_MODE) {
+            if ($this->extensionConfiguration['accessToken'] !== ''
+                && $this->extensionConfiguration['accessTokenSecret'] !== ''
+            ) {
+                return true;
+            }
+
+            return false;
+        }
+
+        if ($this->extensionConfiguration['accessToken'] === ''
+            || $this->extensionConfiguration['refreshToken'] === ''
+        ) {
+            return false;
+        }
+
+        if ($this->extensionConfiguration['expires'] > 0
```

Comment:
Can be simplified:
`return $this->extensionConfiguration['expires'] > 0 && $this->extensionConfiguration['expires'] > time()`

Reply:
Okay. thats weird... `time()` should always be greater than `0` so `return $this->extensionConfiguration['expires'] > time()` should be sufficient.